### DISCUSSION TITLE
Release 1.2.3

### DISCUSSION THREAD

**Discussion by ThomasWaldmann** [borgbackup/borg#7225] [General]
## borgbackup 1.2.3 release

Short changelog summary (changes since borg 1.1):

https://www.borgbackup.org/releases/borg-1.2.html

Long changelog (including upgrade and compatibility notes):

https://github.com/borgbackup/borg/blob/1.2.3/docs/changes.rst#version-123-2022-12-24

### Installation

If you use pip to install this, use: pip install pkgconfig ; pip install "borgbackup==1.2.3"

For other installation methods and more details, please see: https://borgbackup.org/

Note: macOS binaries will come later.

### Feedback

<hr /><em>This discussion was created from the release <a href='https://github.com/borgbackup/borg/releases/tag/1.2.3'>Release 1.2.3</a>.</em>

**Comment by perguth** [comment]
Christmas release, yay =)

**Comment by awgcooper** [comment]
Thanks, as ever, for your serious dedication.

Also, I'm looking forward to 2023 being a borg2 year!
